internal/context/context.go

```diff
@@ -43,6 +43,10 @@ func Init() {
 	nssfContext.NrfUri = fmt.Sprintf("%s://%s:%d", models.UriScheme_HTTPS, nssfContext.RegisterIPv4, 29510)
 }
 
+type NFContext interface {
+	AuthorizationCheck(token, serviceName string) error
+}
+
```

Comment:
add `var _ NFContext = &NSSFContext{}`

Reply:
What is this for?

Reply:
Please see: https://github.com/ianchen0119/uber_go_guide_tw?tab=readme-ov-file#%E4%BB%8B%E9%9D%A2%E5%90%88%E7%90%86%E6%80%A7%E9%A9%97%E8%AD%89